_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Add a Log Drain to your App
+nav: Log Drain
+modified_at: 2018-06-15 10:00:00
+tags: Papertrail Logentries Logmatic Graylog logs integration
```

Comment:
tags are not used ATM. Theses names should appear explicitely in the "real" content of the page.